mem-reserve

```diff
@@ -208,11 +224,64 @@ def parse_size(s):
                 value *= 1024
         return value
 
+def add_suffix(s):
+
+        p_val = 1024 * 1024 * 1024 * 1024 * 1024
+        t_val = 1024 * 1024 * 1024 * 1024
+        g_val = 1024 * 1024 * 1024
+        m_val = 1024 * 1024
+        k_val = 1024
+
+        value = float(s)
+
+        if s / p_val >= 1:
```

Comment:
Right shift by the next-lower range, in integer math, and assign a label if remainer > 0.

Just a suggestion, what you have works.